arangod/GeneralServer/HttpCommTask.cpp

```diff
@@ -316,34 +324,48 @@ bool HttpCommTask<T>::readCallback(asio_ns::error_code ec) {
 }
 
 namespace {
-  static constexpr const char* vst10 = "VST/1.0\r\n\r\n";
-  static constexpr const char* vst11 = "VST/1.1\r\n\r\n";
+static constexpr const char* vst10 = "VST/1.0\r\n\r\n";
+static constexpr const char* vst11 = "VST/1.1\r\n\r\n";
```

Comment:
How about something like this:
```cpp
static constexpr unsigned vstHeaderSize = 11;
// For the header strings we have to add 1 because of the terminating zero byte.
static constexpr char vst10[vstHeaderSize + 1] = "VST/1.0\r\n\r\n";
```
This way you can ensure that the string is not larger than 11 bytes, and you can use a named constant instead of a magic number.

Reply:
I don't think it matters that much here

Reply:
I just noted that the number 11 is used rather often, and I tend to introduce a named constant if I happen to use a "magic constant" more than twice. But this is mainly personal taste. :wink: